bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -678,11 +679,28 @@ public ClientConfiguration setReadEntryTimeout(int timeout) {
     }
 
     /**
-     * Get the interval between successive executions of the PerChannelBookieClient's
-     * TimeoutTask. This value is in milliseconds. Every X milliseconds, the timeout task
-     * will be executed and it will error out entries that have timed out.
+     * Get the interval between successive executions of the operation timeout monitor. This value is in seconds. Every
+     * X seconds, the timeout monitor will be executed and it will error out entries that have timed out.
+     *
+     * @return the interval at which request timeouts will be checked
+     */
+    public long getTimeoutMonitorIntervalSec() {
+        return getLong(TIMEOUT_MONITOR_INTERVAL_SEC,
+                Math.max(Math.min(getAddEntryTimeout(), getReadEntryTimeout()) / 2, 1));
```

Comment:
This setting is also used for quorum timeout. so I think the calculation of the default value should be changed, no?

Reply:
I'll add it, though the quorum timeout shouldn't be less than add entry timeout.